charts/warpstream-agent/README.md

```diff
@@ -464,6 +464,84 @@ kafkaService:
   port: 16500
 ```
 
+### Using a host port for the Kafka port
+
+In some network setups it may be desired to expose the WarpStream Agent as a hostPort on the Kubernetes node.
+
+Using a host port does add lots of complexity to a WarpStream deployment and is only recommended for environments that absolutely 
+require it.
+
+To do this set the following values:
+
+```yaml
+hostPortKafka: 16500
+```
+
+This alone however will not give the desired result as the WarpStream Agents have to be told about the port and to advertise the 
+Kubernetes node address.
+
+A basic setup would be the following:
+
+```yaml
+extraEnv:
```

Comment:
do you do all this automatically when they set that? or do they still have to do it? If so you should probably document its not automatic